update-prices.bash

```diff
@@ -19,43 +19,29 @@ then
     exit 1
 fi
 
-function update_db {
-    db_file="$1"
-    search_exp="$2"
-
-    if [[ -z $db_file ]] || [[ -z $search_exp ]]
-    then
-        echo "Missing arguments, usage: update_db <db-file> <search-exp>"
-        exit 1
+# Define a regex pattern to match the "epoch" line
+EPOCH_PATTERN="^epoch [0-9]+$"
+# Set the flag to start processing lines after the epoch line is found
+START_PROCESSING=false
+
+# Read the input file line by line
+while IFS= read -r line; do
+  if [[ $line =~ $EPOCH_PATTERN ]]; then
+    START_PROCESSING=true
+    continue
+  fi
+  
+  if $START_PROCESSING; then
+    pool=$(echo $line | awk '{print $1}')
```

Comment:
jftr this means that named pools must not contain spaces - might be worth commenting about that in the list of known ones